client/tailscale/localclient.go

```diff
@@ -1028,6 +1028,25 @@ func (lc *LocalClient) DebugSetExpireIn(ctx context.Context, d time.Duration) er
 	return err
 }
 
+// StreamDebugCapture streams a pcap-formatted packet capture.
+//
+// The provided context does not determine the lifetime of the
+// returned io.ReadCloser.
+func (lc *LocalClient) StreamDebugCapture(ctx context.Context) (io.ReadCloser, error) {
+	req, err := http.NewRequestWithContext(ctx, "POST", "http://"+apitype.LocalAPIHost+"/localapi/v0/debug-capture", nil)
+	if err != nil {
+		return nil, err
+	}
+	res, err := lc.doLocalRequestNiceError(req)
+	if err != nil {
+		return nil, err
+	}
+	if res.StatusCode != 200 {
+		return nil, errors.New(res.Status)
```

Comment:
missing res.Body.Close

but isn't there an unexported LocalClient method that does this already? or does doLocalRequestNiceError do that?

Reply:
Done. Poking around, `doLocalRequestNiceError` only handles 403, bad connections, and version mismatches, so we still need this.